.github/workflows/deploy-docs.yaml

```diff
@@ -0,0 +1,32 @@
+name: Deploy docs
+
+# 1. Tag and Release - on merge of v*-version-release
+#    * Tag current version
+#    * Release just tagged version
+# 2. Package and Publish - on published release (from #1)
+#    * Build wheel
+#    * Publish to pypi
+# 3. Deploy docs - on published release (from #1)
+#    * pip install geoips
+#    * pip install plugin repo
+#    * build docs with geoips/docs/build_docs.sh
+#    * deploy docs with geoips/docs/deploy_pages.sh
```

Comment:
This isn't important for this PR, but I'd like to add one more sentence to this to describe the different steps as a whole. Something like "This file is a part of a suite of workflows that run in the following order". Then also add the filenames to the descriptions for each step.

Reply:
The files that are synced to other repos live in the geoips_ci/sync/.github/workflows folder.  It needs to also live in geoips_ci/.github/workflows in order to actually run in the geoips_ci repo. It is kind of annoying that we can't organize workflows in the .github/workflows folder at all, it would be nice to have subdirectories, but I hacked together an organizational solution by naming reusable workflows as reusable-*.yaml, and things implemented as organization rulesets with ruleset-*.yaml.  Everything else is a locally defined workflow.

Reply:
Alright, I'll leave this alone for now, but I think we should give some thought to the filenames here so that things organize in a more reasonable way. Right now, it's hard to tell what files are used in what part of the CI.